controls/radchartview/series/cartesianchart-series/bar-series/barseries.md

```diff
@@ -12,12 +12,19 @@ position: 0
 
 This series is visualized on the screen as separate rectangles representing each of the __DataPoints__.      
 
+* [Declaratively Defined Series](#declaratively-defined-series)
+* [Properties](#properties)
+* [Data Binding](#data-binding)
+* [Features](#features)
+* [Styling the Series](#styling-the-series)
+
+
 ## Declaratively defined series
 
 You can use the following definition to display a simple BarSeries

```

Comment:
You can replace this sentence with the following one.

"_You can use the definition from __Example 1__ to display an BarSeries._"